File: app/src/main/java/dev/ragnarok/fenrir/model/VideoAlbumCriteria.java
package dev.ragnarok.fenrir.model;

import dev.ragnarok.fenrir.db.DatabaseIdRange;
import dev.ragnarok.fenrir.model.criteria.Criteria;


public class VideoAlbumCriteria extends Criteria {

    private final int accountId;

    private final int ownerId;

    private DatabaseIdRange range;

    public VideoAlbumCriteria(int accountId, int ownerId) {
        this.accountId = accountId;
        this.ownerId = ownerId;
    }

    public int getOwnerId() {
        return ownerId;
    }

    public int getAccountId() {
        return accountId;
    }

    public DatabaseIdRange getRange() {
        return range;
    }

    public VideoAlbumCriteria setRange(DatabaseIdRange range) {
        this.range = range;
        return this;
    }
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/model/VideoAlbumCriteria.kt
package dev.ragnarok.fenrir.model

import dev.ragnarok.fenrir.db.DatabaseIdRange
import dev.ragnarok.fenrir.model.criteria.Criteria

class VideoAlbumCriteria(private val accountId: Long, private val ownerId: Long) : Criteria() {
    private var range: DatabaseIdRange? = null
    fun getOwnerId(): Long {
        return ownerId
    }

    fun getAccountId(): Long {
        return accountId
    }

    fun getRange(): DatabaseIdRange? {
        return range
    }

    fun setRange(range: DatabaseIdRange?): VideoAlbumCriteria {
        this.range = range
        return this
    }
}